test/prover/Secp256k1ProverV1.test.ts

```diff
@@ -129,19 +170,73 @@ describe('Secp256k1ProverV1', () => {
       );
     });
 
-    it('rejects batch with lower batch height', async () => {
+    it('rejects batch with zero height', async () => {
```

Comment:
Is this description correct? Theoretically we could now instantiate a prover with batch 50 and submit batch 0, provided that the validator set is mostly the same?

Reply:
It was more about reposting the initial batch (as it is not populated through the `postBatch` function.